2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2037 @@
+academic faculty (“Mittelbau”) in Germany, as well as the already
+mentioned lack of recognition for efforts concerning research software
+for faculty appointments within domain sciences.
+
+In order to develop sustainable software, researchers need to have the
+**skills and expertise** to build software that is easy to maintain and
+extend (Carver, Hong, and Thiruvathukal 2016). However, most of the
+researchers are self-taught developers (Wilson et al. 2014; Philippe et
+al. 2019). Ideally, these skills have to be built into the domain
+science curricula, which could generally be done in two different ways
+(or a combination of them). One obvious solution attempt are additional
+courses that focus on these topics. The main challenge here is to decide
+which other topic(s) to possibly drop due to the limited volume of any
+given curriculum. A different approach is to incorporate
+software-related topics into existing domain science courses. While this
+would provide the benefit of show-casing the usage of specific software
+skills directly within the domain science, the challenge here is the
+amount of work necessary to change existing lecture material, let alone
+the need of the lecturers to acquire those skills themselves in the
+first place.
+
+As long as the necessary software skills within domain sciences are not
+yet wide-spread, building a network from those that have acquired
+relevant skills is difficult. **Community** efforts, that concentrate on
+questions regarding research software, can help to fill this gap.
+However, since research software is such an interdisciplinary topic, it
+is hard to get recognition and find funding within any specific
+discipline. As a result, existing communities often have to rely heavily
+on volunteers. This is challenging, because despite benefits to domain
+science, volunteers hardly receive recognition for their work “back
+home”, i.e., within their domain, underlining again the importance of
+our first demand.
+
+Recommendations
+---------------
+
+Increasing **recognition and awareness** is a challenge that calls for
+both immediate action and perseverance. Nevertheless, some measures will
+likely show positive effects comparatively soon:
+
+Similarly to plans for research data management, funding agencies should
+request that applicants include considerations about how software
+developed in a project can be sustained beyond the end of the funded
+project. A follow up on these plans during and after the project
+lifetime, i.e., a dedicated software management plan, is crucial.
+
+Another recommendation is aimed at decision makers concerning
+recruitment for academic positions: broaden the definition of research
+impact beyond traditional scientific publications to also include other
+impactful results. Not all researchers thinking of themselves as RSEs
+pursue a faculty position as their main career goal. However, permanent
+non-professorial academic faculty positions are rare within the German
```

Comment:
```suggestion
academic non-faculty positions are rare within the German
```

Reply:
What should be meant here is permanent, academic positions other than a professorship (faculty).